doc/build-windows-mingw.md

```diff
@@ -0,0 +1,240 @@
+
+Code:
+
+cd /c/deps/qrencode-3.4.4
+
+LIBS="../libpng-1.6.16/.libs/libpng.a ../../mingw32/i686-w64-mingw32/lib/libz.a" \
+png_CFLAGS="-I../libpng-1.6.16" \
+png_LIBS="-L../libpng-1.6.16/.libs" \
+configure --enable-static --disable-shared --without-tools
+
+make
+
+
+
+2.7 Qt 5 libraries:
+Qt must be configured with ssl and zlib support.
+Download and unpack Qt base and tools sources:
+http://download.qt-project.org/official_releases/qt/5.3/5.3.2/submodules/qtbase-opensource-src-5.3.2.7z
+http://download.qt-project.org/official_releases/qt/5.3/5.3.2/submodules/qttools-opensource-src-5.3.2.7z
+Then from a windows command prompt (note that the following assumes qtbase has been unpacked to C:\Qt\5.3.2 and qttools have been unpacked to C:\Qt\qttools-opensource-src-5.3.2):
+
+Code:
+
+set INCLUDE=C:\deps\libpng-1.6.16;C:\deps\openssl-1.0.1l\include
+set LIB=C:\deps\libpng-1.6.16\.libs;C:\deps\openssl-1.0.1l
+
+cd C:\Qt\5.3.2
+configure.bat -release -opensource -confirm-license -static -make libs -no-sql-sqlite -no-opengl -system-zlib -qt-pcre -no-icu -no-gif -system-libpng -no-libjpeg -no-freetype -no-angle -no-vcproj -openssl -no-dbus -no-audio-backend -no-wmf-backend -no-qml-debug
+mingw32-make -j4
+
+set PATH=%PATH%;C:\Qt\5.3.2\bin
+set PATH=%PATH%;C:\Qt\qttools-opensource-src-5.3.2
+
+cd C:\Qt\qttools-opensource-src-5.3.2
+qmake qttools.pro
+mingw32-make -j4
+Note: consider using -j switch with mingw32-make to speed up compilation process. On a quad core -j4 or -j5 should give the best results.
+
+
+then 
+
+make a BOOST_ROOT environment variable.  best to place it in your mysys.bat file which should
+be on your desktop now.  that way it's there whenever you startup the shell
+
+
+====================================================================================================
+Now you're all setup and it's time for the fun
+
+Build bitcoin
+-------------
+
+As of 12.1 package manager does not work on windows so you need to
+reference pkg.m4 and add the following entry at the top of configure.ac
+
+# This m4 will only be used if a system copy cannot be found. This is helpful
+# on systems where autotools are installed but the pkg-config macros are not in
+# a default location. It is currently used for building on OSX where autotools
+# are preinstalled but pkg-config comes from macports or homebrew. It should
+# probably be removed when building on <= 10.6 is no longer supported.
+m4_include([pkg.m4])
+
+
+then from the mysys shell prompt cd to the root of the source code directory
+and type the following command:
+
+./autogen.sh
+
+Once complete, and also, from the shell prompt enter the following:
+
+CPPFLAGS="-I/c/deps/db-4.8.30.NC/build_unix \
+-I/c/deps/openssl-1.0.1k/include \
+-I/c/deps/libevent-2.0.22/include \
+-I/c/deps \
+-I/c/deps/protobuf-2.6.1/src \
+-I/c/deps/libpng-1.6.16 \
+-I/c/deps/qrencode-3.4.4 \
+-I/c/deps/gmp-6.0.0" \
```

Comment:
This is the first i'm seeing of this dependency, are we missing a section above, of should this not be here?